tiatoolbox/models/classification/patch_predictor.py

```diff
@@ -266,128 +349,331 @@ def predict(
             shuffle=False,
         )
 
-        pbar = tqdm.tqdm(
-            total=int(len(dataloader)), leave=True, ncols=80, ascii=True, position=0
-        )
+        if self.verbose:
+            pbar = tqdm.tqdm(
+                total=int(len(dataloader)), leave=True, ncols=80, ascii=True, position=0
+            )
 
-        model = misc.model_to(on_gpu=on_gpu, model=self.model)
+        # use external for testing
+        model = misc.model_to(on_gpu, self.model)
 
-        all_output = {}
-        predictions_output = []
-        probabilities_output = []
-        labels_output = []
+        cum_output = {
```

Comment:
can we call this `cummulative_output`

Reply:
previously i used that, then I think it becomes too long later